common/libs/values.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import Big from 'big.js';
+import { toWei } from 'libs/units';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
Argument 'v3 has it' only applies when we actually take something from v3 as is. I see no reason to introduce new deps in v4, esp for that case.